webapp/src/index.js

```diff
@@ -19,6 +19,10 @@ export default class Plugin {
             (channelId) => {
                 store.dispatch(openMeetingSettingsModal(channelId));
             });
+
+        registry.registerPostDropdownMenuAction('Re-queue', (postId) => {
+            store.dispatch(requeueItem(postId));
```

Comment:
We need to use `post.props` to:

- declare a given post was made with the agenda plugin
- and check here to see if we should render the `Re-queue` item in the post dropdown menu